.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 
 before_script:
 - pip install tox
+- sed -i 's/\/opt/\/home\/travis\/build\/rcbops/g' ansible-role-requirements.yml
```

Comment:
might it not be cleaner to leave the 'dest' out of the ansible-role-requirements file and use -p when installing with ansible-galaxy to set the role install destination? Then we don't need all the sed shenanigans.